site/docs/v1/tech/guides/configuring-application-specific-email-templates.adoc

```diff
@@ -0,0 +1,112 @@
+
+=== Configuring Application Email Templates
+
+Finally, you'll override your generic Tenant Email Template you just configured with a custom application specific template for a fictitious company called Pied Piper.
+
+Navigate to [breadcrumb]#Customizations -> Emails Templates# and click the blue edit icon for the `Forgot Password` email template and copy the `HTML Template` `Text Template` sections into a notepad.
+
+Create a new email template by going to [breadcrumb]#Customizations -> Emails Templates# and click the green + icon.
+Fill out the `Name`, `Default Subject`, `From Email`, `Default from Name` fields.
+Paste your default template from above into the `HTML Template`  and `Text Template` sections. Feel free to edit these templates based on your application.
+
+Save your template by clicking the blue save icon in the top right corner.
+
+Now configure the email template by navigating to [breadcrumb]#Applications -> Your Application -> click the green edit icon -> Email -> Templates -> Forgot password# and select your application email template - this option will have the same name you populated the `Name` field with from above. Click the blue save icon in the top right corner.
+
+=== Test the Custom Template
+
+For our demo application Pied Piper, we'll call the link:/docs/v1/tech/apis/users#start-forgot-password-workflow[Start Forgot Password Workflow] for a user. You can also use the Login URL, as previously demonstrated.
+```shell
+curl --request POST \
+    https://YOUR_FUSIONAUTH_INSTANCE/api/user/forgot-password \
+  --header 'Authorization: YOUR_API_KEY' \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "applicationId": "85a03867-dccf-4882-adde-1a79aeec50df",
+    "loginId": "dinesh@piedpiper.com",
+     "sendForgotPasswordEmail": true
+}'
+```
+
+Modify the curl command for your `applicationId`, `YOUR_API_KEY`, `YOUR_FUSIONAUTH_INSTANCE`, and `loginId` with the email address of the user you wish to send the email to. Remember this user must be registered for the application before the email can be sent.
+Once successfully executed, you will see your application specific email sent to the above user's email address.
+
+=== Override Email Template using the Hosted Login Page
+
+In addition to calling the link:/docs/v1/tech/apis/users#start-forgot-password-workflow[Start Forgot Password Workflow] API, you can also append the query string parameter to your `Forgot Password` link on the FusionAuth link:/docs/v1/tech/core-concepts/integration-points#hosted-login-pages[hosted login screen].
+Grab your Login URL under [breadcrumb]#Applications# by clicking the green magnifying glass icon for your application and copying the [uielement]#Login URL#. Append `&applicationId=YOUR_APPLICATION_ID` to your Login URL, replacing `YOUR_APPLICATION_ID` with your [field]#applicationId#. Enter your email address and submit the form. You should again see an application specific Forgot Password email in your user's inbox.
```

Comment:
The login url? Don't you need to do something with the forgot password url? Did you mean to say "go to the login url page, then copy the forgot password link"? Or do I totally miss this?

I'd probably drop this whole section unless this is behavior we are willing to commit to supporting in the future.

Reply:
I tested both the Login URL and the Forgot Password URL and they both fire the application-specific email when the `applicationId` is provided. I believe this was part of the original purpose of the guide: if you provide an`applicationId` for action X for an API or a hosted page, we'll send an application-specific email if configured. I'll instruct them to append the `applicationId` to the Forgot Password URL. Since most folks use the hosted pages, I think it's important to include these instructions. However, if we just used the clientId to check if an application-specific email was configured, this would save the manual work, and provide a better user experience.

Reply:
Remove this section 👍